comet/services/anime.py

```
@@ -225,13 +266,28 @@ async def _refresh_from_remote(
+                    "Downloading anime mapping (Source 3/3: Kitsu-IMDB Episode Offsets)...",
+                )
+                async with session.get(self._kitsu_imdb_url) as response_kitsu:
+                    if response_kitsu.status != 200:
+                        logger.warning(
+                            f"Failed to load Kitsu-IMDB mapping: HTTP {response_kitsu.status}"
+                        )
+                        return False
+                    data_kitsu_imdb = orjson.loads(await response_kitsu.read())
+
                 anime_list = data_aod.get("data", [])
                 total_entries, total_fribb = await self._persist_mapping(
```

Ruff (0.14.10)

Unpacked variable `total_fribb` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)